README.md

```diff
@@ -66,7 +66,60 @@ By definition, members work on their respective squad, although they are free to
 
 ## 4. Release process
 
-// Work in progress ⚠️
+### Release engineer as a role
+
+  The release engineer responsibilities are, but not limited to:
+
+  1. Own the entire release process (step-by-step).
+  2. Provide visibility, at all stages, of the release to the wider audience (e.g. squad, tribe and iOS chapter lead).
+  3. Collaborate with QA by:
+    1. Providing visibility to potential blockers to the wider audience.
```

Comment:
This indentation doesn't look like it's working unfortunately 
<img width="953" alt="screen shot 2018-10-22 at 16 13 20" src="https://user-images.githubusercontent.com/773004/47300869-76c3c980-d615-11e8-9766-8ef60ee738e8.png">

Reply:
Fixed.